Sources/MapboxMaps/Annotations/Generated/PointAnnotationManager.swift

```diff
@@ -198,7 +209,7 @@ public class PointAnnotationManager: AnnotationManagerInternal {
         }
         needsSyncSourceAndLayer = false
 
-        let newImages = Set(annotations.compactMap(\.image))
+        let newImages = Set(annotations.compactMap(\.image) + [annotationBeingDragged?.image].compactMap { $0 })
```

Comment:
nit: imho this looks cleaner
```suggestion
        let newImages = Set(annotations.compactMap(\.image) + [annotationBeingDragged].compactMap(\.?.image))
```